### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by jdswinbank**

### CODE REVIEW

File: DM_Applications_Design.tex [hunk 1]
```diff
@@ -353,7 +354,7 @@ \subsubsection{Science Data Quality Analysis Pipeline and Toolkit}
...
+    \item {\bf Level 1 QA} assesses the system status and data quality in real time during commissioning and operations. Its main difference from other observatory, telescope, and camera status reporting tools will be heavy reliance on the massive science imaging data stream (in addition to various telemetry and metadata generated by the subsystems). This level is tasked with nightly reporting of the overall data quality, including the nightly data products (difference images and transient source event stream) and calibration products. Real-time information about observing conditions, such as sky brightness, transparency, seeing, and about the system performance, such as the achieved faint limit, will be delivered by Level 1 QA\@. Because the actual science data stream will be analyzed, Level 1 QA tools will be in a good position to discover and characterize subtle deterioration in system performance that might not be easily caught by tools employed by the telescope and the camera subsystems for self-reporting purposes.
```
Comment (ktlim) [inline DM_Applications_Design.tex]: What's the "\@" added here?

File: DM_Applications_Design.tex [hunk 2]
```diff
@@ -315,21 +317,20 @@ \subsubsection{Level 2 Pipelines Overview}
...
+    \item \emph{Coadd source deblending and characterization}. The next stage in the pipeline will decompose the \CoaddSources into a set of individual astronomical sources which is consistent across all bands, a process known as \emph{deblending}. The deblender may make use of the catalogs of \Sources and \CoaddSources, catalogs of \DIASources, \DIAObjects and \SSObjects detected on difference images, and objects from external. The deblended objects will then be characterized by measuring their positions, shapes and fluxes on the coadded images and by fitting galaxy models. This functionality is contained within the {\bf \emph{Object Detection and Deblending}} system (\wbsDetDeblend) and the {\bf \emph{Object Characterization Pipeline}} (\wbsObjChar).
```
Comment (ktlim) [inline DM_Applications_Design.tex]: "objects from external" _catalogs_; the last word got trimmed.

Comment (jdswinbank) [inline DM_Applications_Design.tex]: Thanks; fixed.

(I saw a comment about `\@` go by, but don't now see it. For the record, it's necessary to get the correct spacing when a sentence ends with a capital letter.)

File: DM_Applications_Design.tex [hunk 3]
```diff
@@ -220,34 +222,34 @@ \subsection{Science Pipelines Overview}
...
-        \item {\bf \em Image Coaddition Pipeline}, (\wbsCoadd), to generate and characterize coadded images of the sky, as well as create templates for image differencing.
-        \item {\bf \em Deep Detection Pipeline}, (\wbsDeepDet), to detect sources in coadded images of the sky.
-        \item {\bf \em Object Characterization Pipeline}, (\wbsObjChar), to characterize (perform measurements of) astrophysical objects detected in LSST imaging (both in single frames and coadds).
+        \item {\bf \emph{PSF Estimation Pipeline}} (\wbsPSF), to estimate the PSF properties and variation across the focal plane for each visit, to the degree of precision required by the \SRD. Note that the work of this pipeline goes beyond the typical single-CCD PSF estimation present in the SFM pipeline.
+        \item {\bf \emph{Image Coaddition Pipeline}} (\wbsCoadd), to generate and characterize coadded images of the sky, as well as create templates for image differencing.
+        \item {\bf \emph{Object Detection and Deblending}} (\wbsDetDeblend), to detect sources in images of the sky and decompose them individual astronomical objects.
```
Comment (gpdf) [inline DM_Applications_Design.tex:237]: decompose them _into_ individual astronomical objects.

Comment (jdswinbank) [inline DM_Applications_Design.tex:237]: Fixed directly on `master`.

**Pull request merged**